build.sbt

```diff
@@ -5,13 +5,15 @@ import scala.sys.process._
 
 skip in publish := true
 
-lazy val integrationTestSettings: Seq[Def.Setting[_]] = Defaults.itSettings ++ Seq(
+lazy val integrationTestSettings: Seq[Def.Setting[_]] = Project.inConfig(IntegrationTest)(baseAssemblySettings) ++ Defaults.itSettings ++ Seq(
```

Comment:
we need to enable the sbt assembly plugin in the "it" context in order to product the jar (this only actually applies to the support-workers project as we are not aggregating separately)